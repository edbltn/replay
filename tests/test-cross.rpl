strat {A, B};

Strategy s1 = Strategy[2|3|{  state0: A, (_,_) -> state0;
                              state1: A, (_,_) -> state0;
                              state2: A, (_,_) -> state0;}];

Strategy s2 = Strategy[2|3|{  state0: B, (_,_) -> state1;
                              state1: B, (_,_) -> state1;
                              state2: B, (_,_) -> state1;}];

Player p1 = Player[s1];
Player p2 = Player[s2];
p1#0.6#p2;

println(p2.strategy);
