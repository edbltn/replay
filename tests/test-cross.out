State 0: play 0
( 0 0 ) -> state 0
( 0 1 ) -> state 0
( 1 0 ) -> state 0
( 1 1 ) -> state 0
State 1: play 0
( 0 0 ) -> state 0
( 0 1 ) -> state 0
( 1 0 ) -> state 0
( 1 1 ) -> state 1
State 2: play 1
( 0 0 ) -> state 1
( 0 1 ) -> state 1
( 1 0 ) -> state 1
( 1 1 ) -> state 1

